core/src/test/scala/org/broadinstitute/dsde/rawls/RawlsTestUtils.scala

```diff
@@ -1,11 +1,12 @@
 package org.broadinstitute.dsde.rawls
 
-import org.broadinstitute.dsde.rawls.dataaccess.slick.TestDriverComponentWithFlatSpecAndMatchers
+import org.broadinstitute.dsde.rawls.dataaccess.slick.{TestDriverComponent, TestDriverComponentWithFlatSpecAndMatchers}
 import org.broadinstitute.dsde.rawls.model.Workspace
+import org.scalatest.{Matchers, Suite}
 import org.scalatest.exceptions.TestFailedException
 import spray.http.{StatusCode, StatusCodes}
 
-trait WorkspaceTestUtils extends TestDriverComponentWithFlatSpecAndMatchers {
+trait RawlsTestUtils extends Suite with TestDriverComponent with Matchers {
```

Comment:
Some tests had a conflict when mixing in the previous base trait.  Suite is more generic.